ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -589,20 +594,25 @@ describe('RuntimeConfigurationPanel', () => {
 
     // Ensure set the form to something non-standard to start
     await pickMainCpu(wrapper, 8);
-    await pickMainDiskSize(wrapper, MIN_DISK_SIZE_GB + 10);
     await pickComputeType(wrapper, ComputeType.Dataproc);
 
+    await pickMainDiskSize(wrapper, 150);
+
     await pickPreset(wrapper, runtimePresets.generalAnalysis);
+    await enableDetachable(wrapper, true);
 
     await mustClickButton(wrapper, 'Create');
 
     expect(runtimeApiStub.runtime.status).toEqual('Creating');
     expect(runtimeApiStub.runtime.configurationType).toEqual(
-      RuntimeConfigurationType.GeneralAnalysis
-    );
-    expect(runtimeApiStub.runtime.gceConfig).toEqual(
-      runtimePresets.generalAnalysis.runtimeTemplate.gceConfig
+      RuntimeConfigurationType.UserOverride
```

Comment:
We do want this to continue to be General Analysis, and it indicates a real bug in the UI.  To reproduce:
1. Create a Dataproc runtime
2. Update it by choosing "General Analysis"

on Test, the UI allows you to change the Dataproc runtime to a GCE runtime
on Local, it does not, because it attempts to select a Standard disk for the GCE runtime

Reply:
Choosing General Analysis when a workspace has no runtime will also demonstrate this bug

Reply:
Fixed

Reply:
please restore the following check as well, to the original
```
expect(runtimeApiStub.runtime.gceConfig).toEqual(
      runtimePresets.generalAnalysis.runtimeTemplate.gceConfig
    );
```